.gitignore

```diff
@@ -1,4 +1,10 @@
 # Maven projects
+/afs/afs-case/target/
```

Comment:
afs-case contains most than cases... it seems to contain the basic objects (scripts)